src/docs/zksync-era.mdx

```diff
@@ -105,27 +105,27 @@ links:
     | FF | SELFDESTRUCT | | Not supported. <br /><br />Produces a compile-time error with the custom compiler. | Halts execution and sends all Ether balance to the specified address. Registers the account for later deletion if executed in the same transaction as the contract was created.<br /><br />Deprecated with [EIP-6059](https://eips.ethereum.org/EIPS/eip-6049) <Unsupported /> |
     | 35 | CALLDATALOAD | | Gets the input data of the current environment.<br /><br />`offset` greater than 2^32-33 causes execution to panic. | Gets input data of the current environment  <Modified /> |
     | 37 | CALLDATACOPY | |  Copies input data in the current environment to memory.<br /><br />Executes a loop of `calldataload` and `mstore` in order to load and copy the `calldata` into memory. The execution will panic if `2^32-32 + offset %32 < offset + len.` | Copies input data in the current environment to memory. <Modified /> |
     | 38 | CODESIZE | | Gets the size of the code in case it is called on a Runtime code.<br /><br />If called on a deploy code, the returned size is the size of the constructor arguments only (excluding deploy code size). | Gets the size of the code running in the current environment. <Modified /> |
-    | 39 | CODECOPY | | The Rollup does not provide a mechanism to access the bytecode of a contract. <br /><br />Copies only constructor arguments when executed on deploy code.<br /><br />Sets memory to `0`s when executed on runtime code compiled through old EVM codegen.<br /><br />Produces compile-time error in case it is executed on runtime code compiling through Yul codegen. | Copies the code running in the current environment to memory. |
+    | 39 | CODECOPY | | The Rollup does not provide a mechanism to access the bytecode of a contract. <br /><br />Copies only constructor arguments when executed on deploy code.<br /><br />Sets memory to `0`s when executed on runtime code compiled through old EVM codegen.<br /><br />Produces compile-time error in case it is executed on runtime code compiling through Yul codegen. | Copies the code running in the current environment to memory. <Modified /> |
```

Comment:
I think we should tag it as unsupported due to "Produces compile-time error in case it is executed on runtime code compiling through through Yul codegen"